.github/workflows/cwf-operator.yml

```diff
@@ -0,0 +1,93 @@
+---
+name: cwf-operator
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
+    paths:
+      - 'cwf/**'
+      - 'k8s/**'
+  pull_request:
+    branches:
+      - master
+jobs:
+  cwf-operator-precommit:
+    runs-on: ubuntu-latest
+    env:
+      GO111MODULE: on  # yamllint disable-line rule:truthy
+      MAGMA_ROOT: "${{ github.workspace }}"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.15.7'
+      - name: Run golang_before_install.sh script
+        run: ./circleci/golang_before_install.sh
+      - name: Run go mod download with retry
+        uses: nick-invision/retry@v2
+        with:
+          command: cd ${MAGMA_ROOT}/cwf/k8s/cwf_operator && go mod download
+          timeout_minutes: 10
+      - name: Run precommit
+        run: |
+            cd ${MAGMA_ROOT}/cwf/k8s/cwf_operator
+            make -C ${MAGMA_ROOT}/cwf/k8s/cwf_operator precommit
+      # Notify ci channel when failing
+      # Plugin info: https://github.com/marketplace/actions/slack-notify
+      - name: Notify failure to slack
+        if: failure() && github.ref == 'refs/heads/master'
+        uses: rtCamp/action-slack-notify@v2.0.2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_CI }}
+          SLACK_TITLE: "Github action cwf-operator-precommit update failed"
+          SLACK_USERNAME: "Cwf workflow"
+          SLACK_ICON_EMOJI: ":boom:"
+          SLACK_COLOR: "#FF0000"
+  cwf-operator-build:
+    needs: cwf-operator-precommit
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run docker compose build
+        run: |
+             cd ${MAGMA_ROOT}/cwf/k8s/cwf_operator/docker
+             DOCKER_REGISTRY=cwf_ docker-compose build --parallel
```

Comment:
Can you export DOCKER_REGISTRY as a step environment variable?

Reply:
oops. i forgot.. thanks!

Reply:
can you delete it from the run command and add the environment variable only for the step

Reply:
sure!

Reply:
Can you remove the export from the run command?